app/views/users/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+
+<html>
```

Comment:
Not needed.
Read that for a better understanding https://guides.rubyonrails.org/layouts_and_rendering.html

Reply:
DONE